docs/services/app_api.md

```diff
@@ -23,7 +23,7 @@ After the DSP container is running, configure the Deploy daemon in AppAPI admin
 or via OCC CLI:
 
 ```bash
-./scripts/occ.sh nextcloud -- app_api:daemon:register dsp_http "DSP HTTP" docker-install http "nextcloud-appapi-dsp-http" "http://nextcloud.local" --net=master_default --set-default
+./scripts/occ.sh nextcloud -- app_api:daemon:register dsp_http "DSP" docker-install http "appapi-dsp:2375" "https://nextcloud.local" --set-default --haproxy_password some_secure_password --net nextcloud_default
```

Comment:
The docs are not fully up to date yet, juts pasted my command here to keep track of it.

Also this should be this one as there is no compose project name set by default:

```suggestion
./scripts/occ.sh nextcloud -- app_api:daemon:register dsp_http "DSP" docker-install http "appapi-dsp:2375" "https://nextcloud.local" --set-default --haproxy_password some_secure_password --net master_default
```